spacesuit/src/bit_range.rs

```diff
@@ -0,0 +1,26 @@
+/// Represents a usize with value in the range [0,64]
+pub struct BitRange(usize);
+
+impl BitRange {
+    /// Returns BitRange if n is <= 64>
+    /// Otherwise Errors
+    pub fn new(n: usize) -> Option<Self> {
+        if n > 64 {
+            None
+        } else {
+            Some(BitRange(n))
+        }
+    }
+
+    /// max returns a BitRange
+    /// representing a 64-bit usize
```

Comment:
That's weird wording. Maybe state simply "/// Returns 64-bit range"?

Reply:
Yeah, this sounds better